src/frontend/src/components/common/InputText/InputText.tsx

```diff
@@ -88,7 +88,7 @@ const InputText = ({
       )}
       {helperText !== '' && (
         <p className="form-group__p--margin-zero">
-          {helperText}
+          {helperText}&nbsp;
```

Comment:
Maybe we could use a bootstrap class like `pe-1` to space out elements, as opposed to literal spaces.

Reply:
yes, actually I get this from your branch, I just check your pr. I did not change this file

Reply:
I believe you. No clue why I would do this as I dislike seeing it. Will fix in a subsequent PR around commenting.